exchanges/mock/common_test.go

```diff
@@ -105,14 +63,10 @@ func TestDeriveURLValsFromJSON(t *testing.T) {
 	}
```

Comment:
🗻 🥃 Just realised how unusual the above test struct is.
Bonus points if you drive by abstract the type and make it saner.

Also really don't have much idea what's going on with this test - Is it ... ensuring that it doesn't blow up when given ... invalid data ... or ?

Reply:
Okay. This wasn't addressed but nevermind.